.github/workflows/conda.yml

```diff
@@ -1,4 +1,4 @@
-name: Build site
+name: Build site with conda
```

Comment:
+1 for the name change - in fact, maybe we should change it even more since the real purpose of this test is to make sure with conda-managed environments, maybe:

```suggestion
name: Test conda
```

Reply:
I went with a little bit more words as I could misunderstand "test conda" for testing notebooks with conda while this job only sphinx builds